src/main/java/core/util/mapper/GenericMapper.java

```diff
@@ -0,0 +1,12 @@
+package core.util.mapper;
```

Comment:
it is better to move mappers to `service` folder

Reply:
Moved